src/content/1.7/modules/concepts/_index.md

```diff
@@ -5,3 +5,15 @@ chapter: true
 ---
 
 # Extension concepts
+
+## Symfony extension concepts
+
+PrestaShop modules acts as Symfony bundles, extension points are added continuously in each minor version:
+
+| Version  | Symfony features                                     |
+|----------|------------------------------------------------------|
+| 1.7.3    | Twig templates inheritance and Services declaration  |
+| 1.7.4    | Forms customization and Console commands declaration |
```

Comment:
How `Console commands declaration` is different that `Services declaration`?

`Forms customization` is a bit confusing.

Reply:
before, if you declare the commands they weren't registered in the Console.

Reply:
I've updated "Forms customization" to "Configuration forms"